### ISSUE TITLE
Add an example `ronin-payloads build` command builder to the `ronin-payloads show` command

### ISSUE THREAD

**Issue by postmodern** [ronin-rb/ronin-payloads#107]
Add a method which builds an example `ronin-payloads build ...` string for the given payload class. It must list any required params (that don't have default values) as `-p #{param.name}=#{params.name.upcase}`.

## Example

```shell
$ ronin-payloads show shellcode/linux/x86/reverse_shell
```

Would output:

```
...

Usage:

    $ ronin-payloads build shellcode/linux/x86/reverse_shell -p host=HOST -p port=PORT
```

**Comment by postmodern** [comment]
We might also want to add logic to come up with `USAGE` values or example values for the params based on their type/name. For the most part `HOST`, `PORT`, and `URL` will probably be common example param values.

**Comment by postmodern** [comment]
Implemented by 2d62191.
